crates/bevy_render/src/draw.rs

```diff
@@ -123,6 +124,8 @@ pub enum DrawError {
 pub struct DrawContext<'a> {
     pub pipelines: ResMut<'a, Assets<PipelineDescriptor>>,
     pub shaders: ResMut<'a, Assets<Shader>>,
+    pub shader_events: ResMut<'a, Events<AssetEvent<Shader>>>,
```

Comment:
once we move shader updating to its own system, lets move these out of DrawContext (probably into a new Local resource used by the shader update system). I don't think they should live on DrawContext because "shader updating" is orthogonal to "drawing"